### PR TITLE
What is a KrakenCthulhu?

### PR DESCRIPTION
**Pull request by puziip**
Example pull request.

### CODE REVIEW

**Comment by matthewjohnpayne** [comment]
This looks wrong

File: CthulhuNewDemon.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+South Park - Cthulhu is like the Kraken!!
```
Comment (ivalaginja) [inline CthulhuNewDemon.txt:1]: I think you misspelled this name

File: CthulhuNewDemon.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Let's release the Kraken!!
```
Comment (ivalaginja) [inline CthulhuNewDemon.txt:2]: Please remove one exclamation point

**Review by ivalaginja — CHANGES_REQUESTED** [review]
Nice job! Almost there, I would just like to see the two things below changed.

**Review by ivalaginja — APPROVED** [review]

**Pull request merged**